experimental/generation/generator/packages/library/src/mergeAssets.ts

```diff
@@ -739,3 +780,38 @@ async function parseSchemas(schemaName: string, oldPath: string, newPath: string
     await copySingleFile(newPath, mergedPath, schemaName + '.json', newFileList, feedback)
     return {oldPropertySet, newPropertySet}
 }
+
+/**
+ * @description: Get the synonyms of the enum entity.
+ * @param schemaName Name of the .schema file.
+ * @param newPath Path to the folder of the new asset.
+ */
+async function getSynonyms(schemaName: string, newPath: string, locale: string): Promise<Map<string, Set<string>>> {
+    let template = await fs.readFile(ppath.join(newPath, schemaName + '.json'), 'utf8')
+    let newObj = JSON.parse(template)
+    let propertyValueSynonyms = new Map<string, Set<string>>()
+
+    for (let property in newObj['properties']) {
+        let examples = newObj['properties'][property]['$examples']
+        if (examples !== undefined) {
+            let valueExamples
+            if (examples[''][`${property}Value`] !== undefined) {
+                valueExamples = examples[''][`${property}Value`]
+            }else if (examples[locale][`${property}Value`] !== undefined) {
+                valueExamples = examples[locale][`${property}Value`]
```

Comment:
These are backwards.  Locale specific first then fallback.

Reply:
Also you are not checking the global $examples.

---
In reply to: [588570829](https://github.com/microsoft/BotBuilder-Samples/pull/3104#discussion_r588570829) [](ancestors = 588570829)